CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+# Overview
+
+This documents explains the processes and practices recommended for
+contributing enhancements to the Prometheus charm.
+
+- It is recommended that prior to developing enhancements to this charm
+  an issue explaining the use case is created at the official
+  Canonical [github repository](https://github.com/canonical/prometheus-operator).
```

Comment:
"- Generally, before developing enhancements to this charm, you should consider [opening an issue](https://github.com/canonical/prometheus-operator/issues/new) explaining your the use case.

Reply:
Fixed.